cmd/cli/app/profile/table_render.go

```diff
@@ -81,38 +81,33 @@ func NewProfileTable() table.Table {
 // RenderProfileTable renders the profile table
 func RenderProfileTable(p *minderv1.Profile, t table.Table) {
 	// repositories
-	renderEntityRuleSets(minderv1.RepositoryEntity, p.Repository, t)
+	renderProfileRow(minderv1.RepositoryEntity, p.Repository, t)
 
 	// build_environments
-	renderEntityRuleSets(minderv1.BuildEnvironmentEntity, p.BuildEnvironment, t)
+	renderProfileRow(minderv1.BuildEnvironmentEntity, p.BuildEnvironment, t)
 
 	// artifacts
-	renderEntityRuleSets(minderv1.ArtifactEntity, p.Artifact, t)
+	renderProfileRow(minderv1.ArtifactEntity, p.Artifact, t)
 
 	// pull request
-	renderEntityRuleSets(minderv1.PullRequestEntity, p.PullRequest, t)
+	renderProfileRow(minderv1.PullRequestEntity, p.PullRequest, t)
 }
 
-func renderEntityRuleSets(entType minderv1.EntityType, rs []*minderv1.Profile_Rule, t table.Table) {
+func renderProfileRow(entType minderv1.EntityType, rs []*minderv1.Profile_Rule, t table.Table) {
```

Comment:
Some refactoring changes:

1) I renamed the function. I understand the logic of the old name, but I feel like the new name is a little more explicit as to what the is doing
2) I collapsed two small functions into one: the `renderRuleTable` function was only ever used by `renderEntityRuleSets`, and I feel like combining them together makes the code easier to follow.

Open to feedback/comments on this one.